src/content/en/fundamentals/primers/customelements/index.markdown

```diff
@@ -369,6 +373,35 @@ class AppDrawer extends HTMLElement {
 
 Define reactions if/when it make senses. If your element is sufficiently complex and opens a connection to IndexedDB in `connectedCallback()`, do the necessary cleanup work in `disconnectedCallback()`. But be careful! You can't rely on your element being removed from the DOM in all circumstances. For example, `disconnectedCallback()` will never be called if the user closes the tab.
 
+**Example:** moving a custom element into another document, observing its `adoptedCallback()`:
+
+{% highlight javascript %}
+function createWindow(srcdoc) {
+  let p = new Promise(resolve => {
+    let f = document.createElement('iframe');
+    f.srcdoc = srcdoc || '';
+    f.onload = e => {
+      resolve(f.contentWindow);
+    };
+    document.body.appendChild(f);
+  });
+  return p;
+}
+
+// 1. Create two iframes, w1 and w2.
+Promise.all([createWindow(), createWindow()])
+  .then(([w1, w2]) => {
+    // 2. Define a custom element in w1.
+    w1.customElements.define('x-adopt', class extends w1.HTMLElement {
+      adoptedCallback() {
+        console.log('Adopted!');
+      }
+    });
+    let a = w1.document.createElement('x-adopt');
+    w2.document.adoptNode(a); // 3. Adopt the custom element into w2.
```

Comment:
There are actually other much more common situations where adoption occurs. Just inserting an element defined in one document into the other will adopt it. So e.g. I'd do `w2.document.body.appendChild(a)` as a good illustration of how this might happen in practice.